src/msix/unpack/InflateStream.cpp

```diff
@@ -45,7 +45,7 @@ namespace MSIX {
         {
             ThrowErrorIfNot(Error::InflateRead,(self->m_compressionObject->GetAvailableSourceSize() == 0), "uninflated bytes overwritten");
             ULONG available = 0;
-            self->m_compressedBuffer = std::make_unique<std::vector<std::uint8_t>>(BufferSize);
+            self->m_compressedBuffer.swap(std::make_unique<std::vector<std::uint8_t>>(BufferSize));

```

Comment:
It is unclear what value this change is providing, unless there are bugs in the STL/compiler being used.